tests/dummy/app/templates/nodes/detail/index.hbs

```diff
@@ -2,18 +2,79 @@
 <div class="panel">
     <h2>Data</h2>
     <p><label>Title:</label> {{model.title}}</p>
+    <p><label>Description: </label> {{model.description}}</p>
     <p><label>Category:</label> {{model.category}}</p>
     <p><label>Date Created:</label> {{moment-format model.dateCreated}}</p>
     <p><label>Date Modified:</label> {{moment-format model.dateModified}}</p>
+    <p><label>Public </label> {{model.public}} </p>
+
     <p>
         <a href="{{model.links.html}}" target="_blank" class="btn btn-primary">
             View on OSF
         </a>
     </p>
+
+    <p><button class="btn btn-danger" {{action "deleteNode" title description category }}>Delete this project</button></p>
     <hr>
-    <h2>Edit this node</h2>
-    Title: {{input value=editedTitle}}
-    <button {{action "editExisting" editedTitle}} class="btn btn-primary">Edit record</button>
+
+    {{#if propertiesVisible}}
+        <button {{action "expandProperties"}}> Close Node Settings </button>
+        <table>
+            <tr>
+                <td><label> Title: </label></td>
+                <td>{{input value=editedTitle}}</td>
+            </tr>
+            <tr>
+                <td><label> Description: </label></td>
+                <td>{{input value=editedDescription}} </td>
+            </tr>
+        </table>
+        <p><label> Category: </label></p>
+        <div>
+            <p>{{radio-button name="category" value="analysis" checked=editedCategory}} Analysis </p>
```

Comment:
Not in scope of this PR, but in the future, it would be nice to break this category selector into a separate component.  I could see this being useful and reusable. 

(and maybe change to a dropdown box)